examples/dataset/person_detection/README.md

```diff
@@ -0,0 +1,87 @@
+# Example Integration: Object Detection (2D)
+
+This example integration uses the [COCO](https://cocodataset.org/#overview) dataset to demonstrate how to test single
```

Comment:
Should we say `coco-2014-val` to be explicit?

Reply:
I think it's fine with just `COCO` -- most of the other stuff we're putting on the public demo seems to mention just COCO in the readme.